src/RollbarServiceProvider.php

```diff
@@ -23,6 +25,18 @@ public function boot()
 
         // Listen to log messages.
         $app['log']->listen(function ($level, $message, $context) use ($app) {
+            
+            $user = Auth::user();
+
+            // log logged in user details
+            if ($user instanceof User) {
```

Comment:
Good point, I've updated this to use `Authenticatable` and optionally try to append other attributes not available on that contract